src/main/webapp/app/exam/participate/summary/exercises/header/exam-result-summary-exercise-card-header.component.html

```diff
@@ -6,7 +6,7 @@ <h5 class="mb-0" [id]="'exercise-group-title-' + exercise.id">
                 <fa-icon [icon]="exerciseInfo!.icon" />
             }
             &nbsp;
-            {{ exercise.exerciseGroup?.title }}
+            {{ exercise.title }}
```

Comment:
This is wrong! For exam exercises, the exercise group title should be shown, NEVER the exercise title:

For a group `Strategy Pattern` there might be different variant that are named differently (`Stragy Pattern 1`, `Stratey Pattern 2`, etc.) This exercise name is for distinction on the side of tutors and instructors. Students should see the exercise group name and should not know which variant they got.